### ISSUE TITLE
Refine LAKES series Serta mattress Discount exclusion because it interferes with QUAIL packages being discounted

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2072]
_Imported from trac ticket 2072.
Created by **wayne**
Opened in trac: Jan 16, 2013, 4:27:40 PM
Last modified in trac: Jan 16, 2013, 4:34:07 PM_
***
_by wayne:_
Scott,

Good morning, why would the 220 discount code not get applied to QUAIL1, QUAIL10, QUAIL2, QUAIL3, and QUAIL4?

Thanks,

Wayne Liburd
___
The standard exclusions have this clause, which you apparently worked on with Michael:

```python
    if orderdetail.packageid in (u'LAKES1',u'LAKES2',u'LAKES3',u'LAKES4') or orderdetail.productid in (u'502350452',u'500368469',u'505350455',u'503368462',u'508350458',u'506368465',u'511350453',u'509368468'):
        return u'This Serta mattress may not be discounted.'
```

'502350452', a twin box, is included there since it is apparently a component of the LAKES series.

**Comment by kentbower** [comment]
_**kbower** trac comment on Jan 16, 2013, 4:29:07 PM:_

Kent,

I would like to keep the mattresses and remove the foundations/box.


Thanks,

Wayne Liburd

**Comment by kentbower** [comment]
_**kbower** trac comment on Jan 16, 2013, 4:34:07 PM:_

(Changeset [[3556]](../commit/846a97311865607e41dfecde0e5d050cc4253241))
* closes #2072 (Kane's Refine LAKES series Serta mattress Discount exclusion because it interferes with QUAIL packages being discounted)

**Comment by kentbower** [comment]
_**kbower** trac status change on Jan 16, 2013, 4:34:07 PM:_ **closed**
